test/libsolidity/semanticTests/viaYul/dirty_memory_read.sol

```diff
@@ -0,0 +1,14 @@
+contract C {
+	function f() public pure returns (uint8 x, bool a) {
+		uint8[1] memory m;
+		assembly {
+			mstore(m, 257)
+		}
+		x = m[0];
```

Comment:
Does it make sense to add the weird case `x == 0x0101`? Or does it not relate?

Reply:
I can add another return value ``b`` and set ``b = (m[0] == 0x0101);`` and expect it to be false?

Reply:
yep